lib/Models/Workflows/TableStylingWorkflow.ts

```diff
@@ -1691,6 +1905,269 @@ export default class TableStylingWorkflow
+            id: `${id}-font`,
+            name: "Font",
+            value: labelTraits.font ?? nullValues.font,
+            setDimensionValue: (stratumId, value) => {
+              labelTraits.setTrait(stratumId, "font", value);
+            }
+          },
+          {
+            type: "select",
+            id: `${id}-style`,
+            name: "Label style",
+            selectedId: labelTraits.style ?? nullValues.style,
+            options: [
+              { id: "FILL", name: "Fill only" },
+              { id: "OUTLINE", name: "Outline only" },
+              { id: "FILL_AND_OUTLINE", name: "Fill and outline" }
+            ],
+            setDimensionValue: (stratumId, value) => {
+              labelTraits.setTrait(stratumId, "style", value);
+            }
+          },
+          {
+            type: "numeric",
+            id: `${id}-scale`,
+            name: "Scale",
+            value: labelTraits.scale ?? nullValues.scale,
+            setDimensionValue: (stratumId, value) => {
+              labelTraits.setTrait(stratumId, "scale", value);
+            }
+          },
+          // Show style traits depending on `style`
+          // three options "FILL", "FILL_AND_OUTLINE" and "OUTLINE"
+          labelTraits.style === "FILL" ||
+          labelTraits.style === "FILL_AND_OUTLINE"
+            ? {
+                type: "color",
+                id: `${id}-fill-color`,
+                name: `Fill color`,
+                value: labelTraits.fillColor ?? nullValues.fillColor,
+                setDimensionValue: (stratumId, value) => {
+                  labelTraits.setTrait(stratumId, "fillColor", value);
+                }
+              }
+            : undefined,
+          labelTraits.style === "OUTLINE" ||
+          labelTraits.style === "FILL_AND_OUTLINE"
+            ? {
+                type: "color",
+                id: `${id}-outline-color`,
+                name: `Outline color`,
+                value: labelTraits.outlineColor ?? nullValues.outlineColor,
+                setDimensionValue: (stratumId, value) => {
+                  labelTraits.setTrait(stratumId, "outlineColor", value);
+                }
+              }
+            : undefined,
+          labelTraits.style === "OUTLINE" ||
+          labelTraits.style === "FILL_AND_OUTLINE"
+            ? {
+                type: "numeric",
+                id: `${id}-outline-width`,
+                name: "Outline width",
+                value: labelTraits.outlineWidth ?? nullValues.outlineWidth,
+                setDimensionValue: (stratumId, value) => {
+                  labelTraits.setTrait(stratumId, "outlineWidth", value);
+                }
+              }
+            : undefined,
+          {
+            type: "numeric",
+            id: `${id}-pixel-offset-x`,
+            name: "Pixel offset X",
+            value: labelTraits.pixelOffset[0] ?? nullValues.pixelOffset[0],
+            setDimensionValue: (stratumId, value) => {
```

Comment:
Cannot set negative values for this by typing in box in UI

Reply:
done!